src/main/java/com/splunk/logging/HttpEventCollectorEventInfo.java

```diff
@@ -18,23 +18,46 @@
  * under the License.
  */
 
+import ch.qos.logback.classic.spi.IThrowableProxy;
```

Comment:
@afinkenstadt This isn't quite ready to go. Using `IThrowableProxy` adds a hard dependency on logback, which is something we definitely don't want (we also need support `java.util.logging` and log4) so forcing people to also depend on logback is not possible.

**Can we refactor this so references to `IThrowableProxy` are isolated to `HttpEventCollectorLogbackAppender.java`? If not, how do you feel about reverting those changes?**
This means removing `IThrowableProxy` from the following files:

* `HttpEventCollectorEventInfo.java`
* `HttpEventCollectorLog4jAppender.java` (unless there's a log4j equivalent)
* `HttpEventCollectorLoggingHandler.java`
* `HttpEventCollectorSender.java`